File: Exemplos M10/App_IntentDigitalHUB_XamarinForms/Elgin_IntentDigitalHUB_XamarinForms-font/E1_Digital_Hub_Xamarin_Forms_M8/IntentServices/Termica/TermicaCommand.cs
using Newtonsoft.Json;

namespace Xamarin_Forms_Intent_Digital_Hub.IntentServices.Termica
{
    abstract class TermicaCommand : IntentDigitalHubCommand
    {
        //O retorno dos comandos de impressora é sempre um int, correspondendo a documentação TERMICA
        [JsonProperty(PropertyName = "resultado")]
        private int resultado;

        protected TermicaCommand(string functionName) : base(functionName, IntentDigitalHubModule.TERMICA) { }

        public int GetResultado()
        {
            return resultado;
        }
    }
}

File: Exemplos SmartPOS/App_IntentDigitalHUB_Flutter/Elgin_IntentDigitalHUB_SmartPOS_Flutter-font/lib/IntentDigitalHubService/TERMICA/termica_command.dart
import 'package:flutter_smartpos/IntentDigitalHubService/intent_digital_hub_module.dart';
import '../intent_digital_hub_command_starter.dart';

abstract class TermicaCommand extends IntentDigitalHubCommand {
  TermicaCommand(String functionName)
      : super(functionName, IntentDigitalHubModule.TERMICA);
}
